### ISSUE TITLE
allow 'this and future events' editing

### ISSUE THREAD

**Issue by gour** [pimutils/khal#524]
Hello,

I use lot of recurring events and it happens that the dates for some of the events (eg. child's activities) do change the termin to another day in a week, so it would be handy if (i)khal could have option to allow changing of 'this and future' events along/instead of "All (past and future)" events, since, imho, it mostly does not matter what did already occur?

**Comment by geier** [comment]
Yes, this really would be good addition, needs some work on the backend part as well though.
